content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,305 @@
+
+## The Undisputed Leader for Azure Infrastructure as Code
+
+Pulumi Azure Native provides direct access to the [Azure Resource Manager API](https://learn.microsoft.com/en-us/rest/api/resources/), making it hands-down the best infrastructure as code solution for Microsoft Azure. Unlike traditional providers that abstract away cloud APIs, Azure Native gives you more than **2x the resource and property coverage** compared to Terraform by directly mapping to Azure's native REST API. This means:
+
+- **Complete API coverage**: 100% of Azure resources available in Azure Resource Manager
+- **Same-day access to new Azure features**: No waiting for provider updates
+- **Advanced property support**: Access to all resource properties, not just a subset
+- **Latest API versions**: Always up-to-date with the newest stable Azure APIs
+
+<!--more-->
+
+And now with V3's dramatic 75% SDK size reduction, we've eliminated the only potential drawback - making our Azure Native provider both the most comprehensive AND the most performant option for managing Azure infrastructure.
+
+![Resource and Property Coverage Comparison](https://www.pulumi.com/images/blog/azure-native-coverage-comparison.png)
+
+These improvements came about through our [RFC process](https://github.com/pulumi/pulumi-azure-native/issues/4004), where we explored various options to balance SDK size with the flexibility needed by advanced users.
+
+## A Slimmer, More Efficient SDK
+
+The most significant improvement in Azure Native V3 is the dramatic 75% reduction in SDK size across all languages. By optimizing the SDK to include only the most commonly used API versions, we've solved one of the most frequent pain points reported by our users:
+
+| Language | Before | After | Reduction | Build Time Before | Build Time After |
+|----------|--------|-------|-----------|-------------------|------------------|
+| .NET     | 77MB | 15MB | 80.5% | 7m 52s | 47s |
+| JS/TS    | 494MB | 87MB | 82.4% | 166s | 25s |
+| Python   | 104.6 MB | 25MB | 76.3% | - | - |
+
+This transformation delivers immediate benefits that every developer will notice:
+
+- Faster IDE performance with reduced load times
+- More responsive language servers for better autocompletion and documentation
+- Reduced memory consumption for resource-constrained development environments
+- Significantly quicker package installation and project bootstrap times
+- Improved CI/CD pipeline performance with faster dependency installation
+
+## Additional Improvements
+
+Azure Native V3 also includes several other enhancements:
+
+- **Reorganized module structure** better aligned with Azure SDK conventions
+- **Improved resource naming** for clearer distinction between similar resources
+- **Enhanced handling of nested properties** to avoid accidental overwriting
+- **Better state management** with the removal of unnecessary `__inputs` fields
```

Comment:
That's true but to be fair won't be noticeable to users, unless maybe they have very large stacks.